Gopkg.toml

```diff
@@ -48,11 +48,31 @@
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.15.4"
+  version = "kubernetes-1.16.3"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.15.4"
+  version = "kubernetes-1.16.3"
+
+[[override]]
```

Comment:
These overrides break the cycle apprclient - k8sclient - operatorkit - e2e-harness - apprclient